entities.bal

```diff
@@ -2,98 +2,90 @@ import ballerina/time;
 
 // Stakeholders (Main Actors)
 type Supplier record {
-  int supplierID = -1;
-  string name;
-  string shortName;
-  string email;
-  string phoneNumber;
+    int supplierID = -1;
+    string name;
+    string shortName;
+    string email;
+    string phoneNumber;
 };
+
 type Beneficiary record {
-  int beneficiaryID = -1;
-  string name;
-  string shortName;
-  string email;
-  string phoneNumber;
+    int beneficiaryID = -1;
+    string name;
+    string shortName;
+    string email;
+    string phoneNumber;
 };
+
 type Donor record {
-  int donarID = -1;
-  string orgName;
-  string orgLink;
-  string email;
-  string phoneNumber;
+    int donarID = -1;
+    string orgName;
+    string orgLink;
+    string email;
+    string phoneNumber;
 };
 
 // Main Types
 type MedicalItem record {
-  int itemID=-1; 
-  string name; 
-  string shortName; 
-  string email;
-  string phoneNumber;
-};
-type RequirementList record {
-  string name;
-  MedicalNeed[] needs = [];
+    int itemID = -1;
+    string name;
+    string 'type;
+    string unit;
 };
+
 type MedicalNeed record {
-  int needID=-1;
-  int itemID;
-  int beneficiaryID; 
-  time:Date period;
-  string urgency;
-  int quantity;
-  Beneficiary? beneficiary = ();
+    int needID = -1;
+    int itemID;
+    int beneficiaryID;
+    time:Date period;
+    string urgency;
+    int neededQuantity;
+    int remainingQuantity;
+    Beneficiary? beneficiary = ();
 };
+
 type Quotation record {
-  int quotationID=-1;
-  int supplierID;
-  int itemID;
-  string brandName;
-  int availableQuantity;
-  time:Date expiryDate; 
-  string regulatoryInfo;
-  decimal unitPrice;
-  Supplier? supplier = ();
-  MedicalItem? medicalItem = ();
+    int quotationID = -1;
+    int supplierID;
+    int itemID;
+    string brandName;
+    int availableQuantity;
+    time:Date period;
+    time:Date expiryDate;
+    string regulatoryInfo;
+    decimal unitPrice;
+    Supplier? supplier = ();
+    MedicalItem? medicalItem = ();
 };
+
 type AidPackage record {
-  int packageID?;
-  string description;
-  string name;
-  string status;
-  AidPackageItem[] aidPackageItems?;
+    int packageID?;
```

Comment:
packageID also can be initialized to -1.

Reply:
Fixed, Thanks